gen/host.go

```diff
@@ -42,29 +42,31 @@ func (gg *Generator) genHostFunctions(g *protogen.GeneratedFile, f *fileInfo) {
 
 	// Define exporting functions
 	g.P(fmt.Sprintf(`
-		func (h %s) Export() map[string]interface{} {
-			return map[string]interface{}{`, structName))
+		// Instantiate a Go-defined module named "env" that exports host functions.
```

Comment:
I changed the generator a little, though it still uses reflection. I'll port this to use non-reflective in a separate change!

Reply:
Cool!